### ISSUE TITLE
Synology: Initializing repo via one-off task doesn't work

### ISSUE THREAD

**Issue by zzottel** [borgbase/docs.borgbase.com#10]
... and will never work, because the BorgBase ssh key has to be added to root's known keys.

I suggest removing that info from https://docs.borgbase.com/setup/synology/ and telling the user that the repository has to be initialized from the command line on the disk station.

**Comment by m3nu** [comment]
This is true and should be mentioned. Personally I also prefer testing stuff in the CLI, but it's still doable without. Specifically, you could add `ssh_command: ssh -o StrictHostKeyChecking=no` to your Borgmatic config.

I don't have a fresh Syno to play with, but if you can confirm this works, I'll update the docs right away.

**Comment by zzottel** [comment]
I don't have a fresh one either, as I used the command line, too. I even moved away from Borgmatic now, as I had my own Borg scripts from my servers, anyway, and Borgmatic confused me more than it helped (especially by not putting borg's stderr output into its own error message when an error occurred).

So, I'm sorry, can't help there anymore.
